common/src/main/java/org/apache/rocketmq/common/protocol/route/TopicRouteData.java

```diff
@@ -27,23 +27,23 @@
 
 public class TopicRouteData extends RemotingSerializable {
     private String orderTopicConf;
-    private List<QueueData> queueDatas;
-    private List<BrokerData> brokerDatas;
+    private List<QueueData> queueDataList;
+    private List<BrokerData> brokerDataList;
```

Comment:
It is modification of protocol, IMO, it's improper for upgrade of server or client.